test/trento/application/projectors/check_result_projector_test.exs

```diff
@@ -17,6 +20,17 @@ defmodule Trento.CheckResultProjectorTest do
     HostChecksExecutionCompleted
   }
 
+  @endpoint TrentoWeb.Endpoint
+
+  setup do
```

Comment:
@CDimonaco 
Maybe we could've have put this `setup` in a macro and use it.
Duplicating over and over again looks tedious hehe

Maybe it is a good exercise to learn about elixir macros

Reply:
@CDimonaco @arbulu89 ExUnit offers a way to do this through [ExUnit.CaseTemplate](https://hexdocs.pm/ex_unit/1.12.3/ExUnit.CaseTemplate.html).
We could define a `Trento.ProjectorCase` and pass the channel option by using the [using](https://hexdocs.pm/ex_unit/1.12.3/ExUnit.CaseTemplate.html#functions) macro to add the shared imports, and then we could access the `tags` in the setup function see `TrentoWeb.ConnCase` for instance.

`use Trento.ProjectorCase, channel: "somechannel"`